src/data/items.h

```diff
@@ -8934,6 +8947,44 @@ const struct Item gItems[] =
         .fieldUseFunc = ItemUseOutOfBattle_CannotUse,
     },
 
+    [ITEM_POKE_FLUTE] =
+    {
+        .name = _("Poké Flute"),
+        .itemId = ITEM_POKE_FLUTE,
+        .price = 0,
+        .description = sPokeFluteDesc,
+        .importance = 1,
+        .pocket = POCKET_KEY_ITEMS,
+        .type = ITEM_USE_BAG_MENU,
+        .fieldUseFunc = ItemUseOutOfBattle_CannotUse,
+    },
+
+    [ITEM_FAME_CHECKER] =
+    {
+        .name = _("FAME CHECKER"),
+        .itemId = ITEM_FAME_CHECKER,
+        .price = 0,
+        .description = sFameCheckerDesc,
+        .importance = 1,
+        .unk19 = 1,
+        .pocket = POCKET_KEY_ITEMS,
+        .type = ITEM_USE_BAG_MENU,
+        .fieldUseFunc = ItemUseOutOfBattle_CannotUse,
+    },
```

Comment:
Make sure to restore data that matches the condition before they were removed from IE. I think the only difference might be the capitalization of names, but it's good to double-check.